[WIP] Add vscode for web support

surajmandalcell opened the pull request:
- Make changes to package and tsconfig files to support web
- Used vscode.workspace.fs to read and write files instead of fs

surajmandalcell commented:
#3 Refer to this issue

collinc777 commented on package.json:
```diff
@@ -1,113 +1,117 @@
-  }
-}
\ No newline at end of file
+{
```
Can you make the diff for this file smaller please?

surajmandalcell replied:
idk what the issue is but I cant get the diff to work properly here, I've copy pasted the files before and after here you can check it out https://www.diffchecker.com/cX1NvPtY/

surajmandalcell committed:
- Make changes to package and tsconfig files to support web
- Used vscode.workspace.fs to read and write files instead of fs
- Fix line endings, this time for sure

surajmandalcell committed:
Merge branch 'main' into vscode-for-web-support
